pros/src/controller.rs

```diff
@@ -94,7 +122,7 @@ impl Controller {
         }
     }
 
-    /// Gets the state of the controller; the joysticks and buttons.
+    /// Gets the current state of the controller in its entirety, including its joysticks and buttons.
```

Comment:
This is worded weirdly. You don't need to specify that the entire state includes buttons  and joysticks.